recipes/onetbb/all/conanfile.py

```diff
@@ -26,17 +27,21 @@ class OneTBBConan(ConanFile):
         "fPIC": [True, False],
         "tbbmalloc": [True, False],
         "tbbproxy": [True, False],
+        "interprocedural_optimization": [True, False],
     }
     default_options = {
         "shared": True,
         "fPIC": True,
         "tbbmalloc": False,
         "tbbproxy": False,
+        "interprocedural_optimization": True,
     }
 
     def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
+        if not (Version(self.version) >= "2021.6.0" and self.options.shared and self.settings.os != "Android"):
```

Comment:
I took this condition from onetbb itself: https://github.com/oneapi-src/oneTBB/blob/c8504c4ec866eb474d323e9df326778f30f9850d/CMakeLists.txt#L187-L190

would be great if you could fix it in upstream

Reply:
Ok i will try to test the upstream project for this. My insights so far were, that this is heavily dependent on version of the ndk. Since the last years the Android NDK was going to a full gcc to clang migration. I am not sure how this should be handled correctly, since it might fail in some ndk versions. But i think it is fine to go for now with your working state and an dedicated search on how to enable IPO for Android later on. It should not block the effort for now.

TLDR: fine for me now ;)